ppb/scenes.py

```diff
@@ -96,16 +96,20 @@ def remove(self, game_object: Hashable) -> None:
 
 
 class BaseScene(Scene, EventMixin):
+    # Background color, in RGB, each channel is 0-255
+    background_color: Sequence[int] = (0, 0, 100)
 
-    def __init__(self, engine, *, background_color: Sequence[int]=(0, 0, 100),
+    def __init__(self, engine, *,
                  container_class: Type=GameObjectCollection,
                  set_up: Callable=None, pixel_ratio: Union[int, float]=80,
                  **kwargs):
         super().__init__(engine)
-        self.background_color = background_color
-        self.background = None
+        for k, v in kwargs.items():
```

Comment:
This has taken some time, but I'm super for this now that I think about it. It means you don't even need to subclass if you don't want to.